app/components/Button/Redesign/Button.jsx

```diff
@@ -0,0 +1,62 @@
+// @flow
```

Comment:
Can we just replace the other button component with this one instead?  We won't be releasing v2 until it's ready anyway.

Reply:
👍